server/main.py

```diff
@@ -108,16 +143,32 @@ def odlc_submit(id_):
     return jsonify(interop.odlc_submit(id_))
 
 
-@app.route("/interop/odlc/load")
+@app.route("/interop/odlc/load", methods=["POST"])
 def odlc_load():
     return jsonify(interop.odlc_load_queue())
 
 
-@app.route("/interop/odlc/save")
+@app.route("/interop/odlc/save", methods=["POST"])
 def odlc_save():
     return jsonify(interop.odlc_save_queue())
 
 
+@app.route("/interop/map/add/<string:name>", methods=["POST"])
```

Comment:
The name is used just as a location for where to save the file, and feels better as a URL parameter. However, the image needs to be sent in base64 encoded, so it can be sent via POST. My idea of sending the name in the get request is because the name is an identifier for a specific map image.